docs/api/ref/GetAgent.md

```diff
@@ -0,0 +1,224 @@
+
+The details persisted conform to the following type:
+
+```ts
+/** Type representing data on the Desktop Agent that an app 
+ *  connected to that is persisted by the getAgent function 
+ *  to be used when re-connecting (after a navigation or  
+ *  refresh event) and to ensure a consistent instanceId.  
+ */ 
+type DesktopAgentDetails = { 
+    /** The type of Desktop Agent connected to. Used to  
+     *  prevent an inadvertent switch to a different agent.*/ 
+    agentType: WebDesktopAgentType,
+
+    /** The URL that was previously sent to the Desktop Agent 
+     * to establish the app's identity.*/ 
+    identityUrl?: string,
+
+    /** The current URL at the time of the last connection to 
+     * a Desktop Agent.*/ 
+    actualUrl?: string,
+
+    /** Optional URL field that should be used to store any 
+     *  URL that was used to connect to a Desktop Agent. URLs 
+     *  may have been provided by a parent window that has since 
+     *  gone away and persisting may allow re-connection in such 
+     *  cases. */ 
+    agentUrl?: string,
+
+    /** The appId that was identified for the application by the 
+     * Desktop Agent.*/ 
+    appId: string,
+
+    /** The instanceId that was issued to the app by the Desktop  
+     * Agent. */ 
+    instanceId: string,
+
+    /** The instanceUuid that was issued to the app. This should be 
+     *  passed when connecting to the Desktop Agent to help  
+     *  identify that this app has connected before and which  
+     *  instance it is, enabling the Desktop Agent to reissue 
+     *  the same instanceId. The instanceUuid should never be shared 
+     *  with other applications and is not available through the 
+     *  FDC3 API, allowing it to be used as a shared secret with 
+     *  the Desktop Agent that issued the associated instanceId.*/ 
+    instanceUuid: string 
+} 
+ 
+/** Enumeration of values used to describe types of web-based 
+ *  Desktop Agent. Each 'type' refers to the means by which 
+ *  a connection to the agent is made and/or an interface to it 
+ *  received. */ 
+enum WebDesktopAgentType { 
+    /** Denotes Desktop Agents that inject the FDC3 interface  
+     *  at `window.fdc3`. */ 
+    PRELOAD = "PRELOAD",
+
+    /** Denotes Desktop Agents that run (or provide an interface) 
+     *  within a parent window or frame, a reference to which  
+     *  will be found at `window.opener`, `window.parent` or 
+     *  `window.parent.opener`. */ 
```

Comment:
Worth adding "or similar" at the end of the sentence after "`window.parent.opener`" given that the algorithm traverses upwards now (instead of simply trying these 3 fixed possibilities, like when this part was originally written)

Reply:
Fixed in be131b655bbc4812c9cfec080fb8ee86ac81cb85
Went with adding an 'etc.'